packages/backend/src/models/whiteboard.ts

```diff
@@ -435,7 +435,7 @@ const whiteboards: Map<Whiteboard['id'], Whiteboard> = new Map();
 export const addWhiteboard = (host: ClientConnection, uuid?: UUID) => {
   const board = new Whiteboard(host, uuid);
   whiteboards.set(board.id, board);
-  console.log('Whiteboard created', board);
+  logger.info(`Whiteboard created: ${board.id}`);
```

Comment:
@wgslr Just tagging here: only the ID is required for logging, or would you like to see more?

Reply:
Hm. I don't have a specific use case. We could also print the client id, but... the ClientConnection objects are  quite volatile, so I don't think it is important.